spec/features/user_edits_a_review_spec.rb

```diff
@@ -10,13 +10,13 @@
     click_on "Edit Review"
 
     fill_in "Title", with: "Edited Title"
-    fill_in "Body", with: "Edited Body"
+    fill_in "Body", with: "#{'y' * 50}"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.